packages/blocks/src/store/process-block-type.js

```diff
@@ -61,7 +61,22 @@ function mergeBlockVariations(
 
 	return result;
 }
+function handleExperimentalBorder( rawSupports ) {
```

Comment:
We'll also need to handle the transformation for Typography at some point (which I worked on in #63401). Would it be worth renaming this to `stabilizeSupports` so that we have one function call, and we can add other stabilized supports to the body of the function in follow-ups?

Reply:
Function name has been updated as you mentioned above.